requirements/base.txt

```diff
@@ -14,7 +14,7 @@ dateparser~=1.1
 requests==2.28.2
 serverlessrepo==0.1.10
 aws_lambda_builders==1.28.0
-tomlkit==0.11.6
+tomlkit==0.7.2
```

Comment:
why downgrading it?

Reply:
I just reverted the upgrade PR https://github.com/aws/aws-sam-cli/pull/4757

Reply:
I changed the PR to upgrade tomlkit to `0.11.7`